_config.yml

```diff
@@ -33,6 +33,9 @@ navigation:
 - text: FAQ
   url: faq/
   internal: true
+- text: Server Name Indication
```

Comment:
Should we put `SNI` in parens? I think the initialism is at least as well known as the name. (The text may be too long with that, dunno)